pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -32,6 +32,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
+	"k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
```

Comment:
please name this ```framework``` if changing the struct name to ```frameworkImpl```

Reply:
done